.github/dependabot.yml

```diff
@@ -10,20 +10,19 @@ updates:
     labels:
       - "dependencies"
       - "go"
-      - "type::chore"
     schedule:
       interval: "weekly"
     groups:
       security:
         update-types:
           - "patch"
           - "minor"
+          - "type::security"
```

Comment:
I don't believe this is the right place for this.

Reply:
yes, I think should be in labels: